# Pick EB installed GOES location
# May 14th, 2018 by B. Hajgato (Free University Brussels - VUB)
--- basemap-1.1.0/setup.py.orig	2017-05-04 22:53:42.000000000 +0200
+++ basemap-1.1.0/setup.py	2018-05-14 22:56:12.370184105 +0200
@@ -38,8 +38,8 @@
     return geos_version
 
 # get location of geos lib from environment variable if it is set.
-if 'GEOS_DIR' in os.environ:
-    GEOS_dir = os.environ.get('GEOS_DIR')
+if 'EBROOTGEOS' in os.environ:
+    GEOS_dir = os.environ.get('EBROOTGEOS')
 else:
 # set GEOS_dir manually here if automatic detection fails.
     GEOS_dir = None
